src/types.ts

```diff
@@ -431,3 +431,22 @@ export type Reload = (
   optHref: DOMString | null | undefined,
   opts: HvComponentOptions,
 ) => void;
+
+export type LogPayload = {
+  data: unknown;
+  message: string;
+  prefix?: string;
+  severity: number;
+  source?: string;
+  timestamp: Date;
+};
+
+export const LOG_SEVERITY = {
+  ERROR: 0,
+  INFO: 30,
+  LOG: 20,
+  WARN: 10,
+};
+Object.freeze(LOG_SEVERITY);
```

Comment:
Are the values set as severity used for any purpose? If not we may make this a string enum. Did you also try to use the ["enum" syntax](https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums) to declare this?

Reply:
Numeric vs string: you can't filter on "less than or equal to LOG" if the values are strings. Enum: I can try that out.

Reply:
I haven't seen this being used - but since there's a finite number of options, explicitly testing for the value might lead to same result?

Reply:
The spacing in-between each value was intended to provide sub-level severity. Example: log("checking something", 19). This would allow us to still see it when we do a filter:

if (severity <== LOG_SEVERITY.LOG){
  // log
}

But would also allow specific filtering:
if (severity === 19){
  // log
}